Template/NLO/SubProcesses/mint_module.f90

```diff
@@ -1482,6 +1482,7 @@ subroutine close_run_zero_res
     average_virtual(0,1:nchans)=0d0
     call write_grids_to_file
     call write_results
+    call regrid_MC_integer
     open (unit=12, file='res.dat',status='unknown')
```

Comment:
otherwise runs on clusters will complain about a missing file (for the runs with zero results only)
see
https://bazaar.launchpad.net/~maddevelopers/mg5amcnlo/3.0.1-lepcoll/revision/1008

Reply:
ok but am I wrong that this has nothing related to the lepton collider case?

Reply:
well, it was added because of an issue related to the inclusion/non-inclusion of photons in the initial state.
I agree that its scope is more general (e.g. running a process generated for hadron colliders, with some subprocess featuring initial-state photons, but with a PDF set in which photon densities are zero)